auth/customers.go

```diff
@@ -9,6 +10,22 @@ import (
 	"github.com/alexellis/derek/types"
 )
 
+const defaultCustomersURL string = "https://raw.githubusercontent.com/alexellis/derek/master/.CUSTOMERS"
+const customersURLEnv string = "customers_url"
+
+func findCustomersURL() string {
+
+	if customURL, exists := os.LookupEnv(customersURLEnv); exists && (len(customURL) > 0) {
+
+		if !strings.HasPrefix(strings.ToLower(customURL), "http") {
```

Comment:
I don't know why we need this prepending of https?

If the customers URL doesn't have a prefix of http/https we should probably throw an error.

Reply:
This I think can be removed though. I'm not sure what the usecase is for it.